porcupine/plugins/filemanager.py

```diff
@@ -122,6 +125,24 @@ def rename(old_path: Path) -> None:
         tab.path = new_path / tab.path.relative_to(old_path)
 
 
+def paste(new_path: Path) -> None:
+    global copy_path
```

Comment:
Globalling here is not needed, because this function only uses the current value of `copy_path`. You need `global` only when modifying. But to make the type checker happy, you need to ensure it is not `None`.

```suggestion
    assert copy_path is not None
```